### ISSUE TITLE
EventEmitter memory leak

### ISSUE THREAD

**Issue by dannav** [minutemailer/react-popup#52]
In componentWillUnmount() event listeners that were placed on the store in componentDidMount() were not removed.

```js
    componentDidMount() {
        Store.on(Constants.SHOW, this.bound.onShow);
        Store.on(Constants.CLOSE, this.bound.onClose);
        Store.on(Constants.REFRESH, this.bound.onRefresh);
    }

    componentWillUnmount() {
        // need to remove Store event listeners
        key.deleteScope('react-popup');
        key.filter = defaultKeyFilter;
    }
```

**Comment by tbleckert** [comment]
Ah, thanks! Will make sure it's added in the next release

**Comment by quannt** [comment]
I want to help on this.  @tbleckert

**Comment by tbleckert** [comment]
Merged in master (thanks to @quannt ). Will be published to npm along with the other v1.0.0 stuff asap.
